settings.php

```diff
@@ -27,12 +27,13 @@
 
 if ($hassiteconfig) {
 
-    $ADMIN->add('tools', new admin_externalpage(
-        'toolexcimer',
+    $report = new admin_externalpage(
+        'tool_excimer_report',
         get_string('pluginname', 'tool_excimer'),
-        "$CFG->wwwroot/$CFG->admin/tool/excimer/index.php",
+        new moodle_url('/admin/tool/excimer/index.php'),
         'moodle/role:manage'
```

Comment:
this shouldn't use the role:manage. Easiest for now is use ''moodle/site:config' but later we might make our own capability